index.js

```diff
@@ -5,20 +5,31 @@ const slacker = require('./slack');
 const NotificationBase = require('screwdriver-notifications-base');
 const hoek = require('hoek');
 
+// This should match what is in https://github.com/screwdriver-cd/data-schema/blob/master/models/build.js#L98
+// https://github.com/screwdriver-cd/ui/blob/master/app/styles/screwdriver-colors.scss
 const COLOR_MAP = {
-    SUCCESS: 'good',
-    FAILURE: 'danger',
     ABORTED: 'danger',
-    // https://github.com/screwdriver-cd/ui/blob/master/app/styles/screwdriver-colors.scss
+    CREATED: '#0b548c', // Using 'sd-info-fg' from https://github.com/screwdriver-cd/ui/blob/master/app/styles/screwdriver-colors.scss
+    FAILURE: 'danger',
+    QUEUED: '#0F69FF',
     RUNNING: '#0F69FF',
-    QUEUED: '#0F69FF'
+    SUCCESS: 'good',
+    BLOCKED: '#ccc', // Using 'sd-light-gray' from https://github.com/screwdriver-cd/ui/blob/master/app/styles/screwdriver-colors.scss
+    UNSTABLE: '#ffd333', // Using 'sd-unstable' from https://github.com/screwdriver-cd/ui/blob/master/app/styles/screwdriver-colors.scss
+    COLLAPSED: '#f2f2f2', // New color. Light grey.
+    FROZEN: '#acd9ff' // Using 'sd-frozen' from https://github.com/screwdriver-cd/ui/blob/master/app/styles/screwdriver-colors.scss
 };
 const STATUSES_MAP = {
-    SUCCESS: ':sunny:',
-    FAILURE: ':umbrella:',
     ABORTED: ':cloud:',
+    CREATED: ':progress:',
```

Comment:
The following `emoji` or slack emoticons does NOT exist for standard slack.

```
:progress: 
```

I typed it in the SD Slack, and its not available, and also searched in 
https://screwdriver-cd.slack.com/customize/emoji?utm_source=in-prod&utm_medium=inprod-customize_link-slack_menu-click

To see if its standard or customized.

Maybe that can be changed for something more standard like `:clock11:`? 
![image](https://user-images.githubusercontent.com/15989893/69387087-8dee4400-0c79-11ea-92fb-cb18557fe6f1.png)

Reply:
:progress: works in my Slack. I tested them all

Reply:
But I can change it ...

Reply:
Done